### ISSUE TITLE
Celery_app typing error on docker-compose

### ISSUE THREAD

**Issue by waybarrios** [GeoNode/geonode#2682]
Please! Fix it Celery service on docker-compose:

Correct way: 

```

celery:
    image: geonode/django
    links:
      - rabbitmq
      - postgres
      - elasticsearch
    command: celery worker --app=geonode.celery_app:app -B -l INFO
```

Instead of: 

```

celery:
    image: geonode/django
    links:
      - rabbitmq
      - postgres
      - elasticsearch
    command: celery worker --app=geonode.celeryapp:app -B -l INFO
```

**Comment by harts-boundless** [comment]
Here is the diff of the two
```8c8
<     command: celery worker --app=geonode.celery_app:app -B -l INFO
---
>     command: celery worker --app=geonode.celeryapp:app -B -l INFO
```

There is indeed a file at `geonode/celery_app.py` but not a `geonode/celeryapp.py`.

**Comment by francbartoli** [comment]
Closed with #2697
